devito/ops/transformer.py

```diff
@@ -221,10 +221,13 @@ def create_ops_fetch(f, name_to_ops_dat, time_upper_bound):
 def create_ops_par_loop(trees, ops_kernel, parameters, block, name_to_ops_dat,
                         accessible_origin, par_to_ops_stencil, dims):
     it_range = []
+    devito_to_ops_indexer = 1
     for tree in trees:
         if isinstance(tree, IterationTree):
-            for bounds in [it.bounds() for it in tree]:
-                it_range.extend(bounds)
+            for it in tree:
+                it_range.extend((it.dimensions[TimeFunction._time_position].extreme_min,
```

Comment:
extending a list with a tuple? 😮

Reply:
why can you not just use ``it.symbolic_max`` etc ? btw , call it ``i``, not ``it``, for homogeneity. We never use `it` for Iterations

Reply:
Well, I was really overthinking it! thanks for the feedback! About the tuple, I was using it as the code was but it doesn't really make a difference, switched it to a list :)